packages/wonder-blocks-modal/src/components/modal-dialog.js

```diff
@@ -48,6 +47,21 @@ type Props = {|
      * Test ID used for e2e testing.
      */
     testId?: string,
+
+    /**
+     * The label for this dialog, if applicable.
+     */
+    "aria-label"?: string,
```

Comment:
**suggestion:** just a reminder about removing this prop (as you already suggested in Slack)